src/config.rs

```diff
@@ -177,7 +181,8 @@ impl Default for Config {
             kafka_send_timeout_ms: 500,
             db_path: "./taskbroker-inflight.sqlite".to_owned(),
             db_write_failure_backoff_ms: 4000,
-            db_insert_batch_size: 1024,
+            db_insert_batch_size_rows: 256,
```

Comment:
This change in defaults will impact existing deployments. We'd need to update the configuration of existing deployments before this could be merged.